src/test/java/io/vlingo/xoom/symbio/store/state/dynamodb/DynamoDBBinaryStateActorTest.java

```diff
@@ -60,36 +56,30 @@ public void setUp() {
+          new DynamoDBDispatcherControlInstantiator(Arrays.asList(dispatchers), dynamodb, new BinaryStateRecordAdapter(), 1000L, 1000L)));
 
-      stateStore = stateStoreProtocol(world, dispatcher /*dispatchers*/, dispatcherControl, dynamodb, createTableInterest);
+      stateStore = stateStoreProtocol(world, dispatchers, dispatcherControl, dynamodb, createTableInterest);
     }
 
     @Override
-    protected StateStore stateStoreProtocol(World world, Dispatcher<Dispatchable<Entry<?>, BinaryState>> dispatcher, DispatcherControl dispatcherControl, AmazonDynamoDBAsync dynamodb, CreateTableInterest interest) {
+    protected StateStore stateStoreProtocol(World world, List<Dispatcher<Dispatchable<Entry<?>, BinaryState>>> dispatchers, DispatcherControl dispatcherControl, AmazonDynamoDBAsync dynamodb, CreateTableInterest interest) {
       return world.actorFor(
         StateStore.class,
         Definition.has(
                 DynamoDBStateActor.class,
-                new DynamoDBStateStoreInstantiator<>(Arrays.asList(dispatcher), dispatcherControl, dynamodb, interest, new BinaryStateRecordAdapter()))
+                new DynamoDBStateStoreInstantiator<>(dispatchers, dispatcherControl, dynamodb, interest, new BinaryStateRecordAdapter()))
       );
     }
 
     @Override
-//    protected void verifyDispatched(List<Dispatcher<Dispatchable<Entry<?>, BinaryState>>> dispatchers, String id, Dispatchable<Entry<?>,BinaryState> dispatchable) {
-//        verify(dispatchers).dispatch(dispatchable);
-//    }
-    protected void verifyDispatched(Dispatcher<Dispatchable<Entry<?>, BinaryState>> dispatcher, String id, Dispatchable<Entry<?>,BinaryState> dispatchable) {
-      verify(dispatcher).dispatch(dispatchable);
-  }
+    protected void verifyDispatched(List<Dispatcher<Dispatchable<Entry<?>, BinaryState>>> dispatchers, String id, Dispatchable<Entry<?>,BinaryState> dispatchable) {
+      dispatchers.forEach(dispatcher -> verify(dispatcher, atLeast(1)).dispatch(dispatchable));
+    }
 
     @Override
-//    protected void verifyDispatched(List<Dispatcher<Dispatchable<Entry<?>, BinaryState>>> dispatchers, String id, BinaryState state) {
-//        verify(dispatchers, timeout(DEFAULT_TIMEOUT)).dispatch(new Dispatchable<>(id, LocalDateTime.now(), state, Collections.emptyList()));
-//    }
-    protected void verifyDispatched(Dispatcher<Dispatchable<Entry<?>, BinaryState>> dispatcher, String id, BinaryState state) {
-      verify(dispatcher, timeout(DEFAULT_TIMEOUT)).dispatch(new Dispatchable<>(id, LocalDateTime.now(), state, Collections.emptyList()));
-  }
+    protected void verifyDispatched(List<Dispatcher<Dispatchable<Entry<?>, BinaryState>>> dispatchers, String id, BinaryState state) {
+      dispatchers.forEach(dispatcher -> verify(dispatcher, timeout(DEFAULT_TIMEOUT).atLeast(1)).dispatch(new Dispatchable<>(id, LocalDateTime.now(), state, Collections.emptyList())));
```

Comment:
@VaughnVernon I am not sure if this is intended, but dispatchers are called more than once by most of tests (twice, with some extreme cases of 7 times).